src/ProcessMaker/Nayra/Contracts/RepositoryInterface.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+namespace ProcessMaker\Nayra\Contracts;
+
+use ProcessMaker\Nayra\Contracts\Bpmn\FlowNodeInterface;
+use ProcessMaker\Nayra\Contracts\Repositories\StorageInterface;
+
+/**
+ * Repository interface to create BPMN objects.
+ *
+ */
+interface RepositoryInterface
```

Comment:
There another interface with the same name in src/ProcessMaker/Nayra/Contracts/Repositories/RepositoryInterface.php
It could be better if they haver different names.